### ISSUE TITLE
Are vector elements zero-initialized on bit_resize()?

### ISSUE THREAD

**Issue by risenberg** [simongog/sdsl-lite#301]
I expect int_vector to zero-initialize new elements when calling bit_resize() with increased size.
I could not find any hint in documentation, so looked up the code. There is a single bits::write_int call in memory_manager::resize and some comments about zeroing around it. But write_int is limited to 64 bits, so in general the new elements will be uninitialized. Want to know whether it is bug or feature.

**Comment by simongog** [comment]
Yes, bit_resize() does not initialize new elements. It's not promised anywhere so it's not a bug ;)

**Comment by risenberg** [comment]
May be specify it in comments? It is a bit counterintuitive, because resize() from STL does initialize.

**Comment by simongog** [comment]
@risenberg, thanks for the comment. I agree that is is counterintuitive. I will modified it.
